modules/nodes-descheduler-configuring-policies.adoc

```diff
@@ -0,0 +1,57 @@
+// Module included in the following assemblies:
+//
+// * nodes/scheduling/nodes-descheduler.adoc
+
+[id="nodes-descheduler-configuring-policies_{context}"]
+= Configuring descheduler policies
+
+You can configure which strategies that the descheduler uses to evict Pods.
+
+// TODO: need to be cluster admin?
+.Prerequisites
+* Cluster administrator privileges.
+
+.Procedure
+
+. Edit the `KubeDescheduler` object:
++
+----
+$ oc edit kubedeschedulers.operator.openshift.io -n openshift-kube-descheduler-operator
+----
++
+// TODO: should we specify `cluster` one? Can you even have more than one?
```

Comment:
yes, `kubedeschedulers.operator.openshift.io` is just the resource type, and by default the CR we create is called `cluster`. Technically you could have more than one but you should not, and maybe we can package in some kind of cluster quota on these

Reply:
Thanks, I'll specify `cluster` then.